astropy/coordinates/earth.py

```diff
@@ -806,4 +808,4 @@ def _to_value(self, unit, equivalencies=[]):
 
 
 # need to do this here at the bottom to avoid circular dependencies
-from .sites import get_builtin_sites, get_downloaded_sites
+from .sites import get_builtin_sites, get_downloaded_sites  # noqa
```

Comment:
I think this is really a left-over. Maybe it can just be removed.

Reply:
If I remove them, `flake8` complains about undefined names, so the import is still used. FYI.

Reply:
Well, clearly my review was just noise! Sounds like it is all good to go, then!